packages/suite/src/views/suite/bridge/index.tsx

```diff
@@ -148,6 +153,11 @@ const InstallBridge = (props: Props) => {
                         id="TR_CURRENTLY_INSTALLED_TREZOR"
                         values={{ version: data.currentVersion }}
                     />
+                    {isDesktop() && (
+                        <BridgeDesktopNote>
```

Comment:
i think this message should be displayed instead of "gathering information loader" (line 164) because now they are visible both
so this component should have 3 conditional views:
- desktop info (only in desktop)
- loading (web only)
- bridge info (web only)

Reply:
Displaying loading and downloads links on desktop is correct imo. As the text hints that updating bridge is an option, providing the links to do so is correct. Loading might happen if the bridge gets killed. So it would be good to have an indicator of some sort.

Reply:
ah right, it makes sense